_sections/home-english.md

```diff
@@ -172,7 +172,9 @@ In general, choice names should be unique within a single-choice list. If two ch
 
 #### Specify other
 
-For multiple-choice questions, surveys often include an option of marking **other** when their answer choice is not listed. Then, they are usually asked to specify the other option. This is possible through XLSForm by including **or_other** after the answer choice list name in the survey worksheet. The choices worksheet stays the same. See below:
+{% include alerts/warning.html content="This shortcut for specifying an **other** choice only works for single-language surveys and selects without **choice_filter**s. It uses the default value of **other** as the choice name that will be visible in collected data, and the default text of \"Specify other\" for the follow-up question displayed to the user. We generally recommend using [relevance](#relevant) to specify your own **other** choice." %}
```

Comment:
Suggestion - make the recommendation more assertive, and the warning more concise

> We recommend using [relevance](#relevant) to specify your own **other** choice. This shortcut only works for selects without translations or **choice_filter**s. It uses English for the "Specify other" choice, which cannot be customized and will be blank for additional languages.